tests/handlers/websocket/test_listeners.py

```diff
@@ -133,6 +158,24 @@ def handler(data: str) -> Dict[str, str]:
         assert ws.receive_json(mode=send_mode) == {"data": "foo"}
 
 
+@pytest.mark.parametrize("send_mode", ["text", "binary"])
+def test_listener_send_with_dto(send_mode: WebSocketMode, mock: MagicMock) -> None:
+    @dataclass
+    class User:
+        name: str
+
+    user_dto = DataclassDTO[User]
```

Comment:
This should probably include some DTO specific things, since serialisation of dataclasses is supported by default, even without DTOs.

Reply:
This was a good catch, cheers!